CHANGES.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## 5.0.0 (Unreleased)
-
+- ios mobile viewport was lessthan 100vh so had to redefine vh units each time 
```

Comment:
Please fix `lessthan` and write `ios` as `iOS`. Also add the ticket number in front and make sure that it's placed in the right position (ordered by ticket number).